### ISSUE TITLE
Cannot find module 'decompress-zip'

### ISSUE THREAD

**Issue by samirchian** [bubenshchykov/ngrok#129]
```> ngrok@github:screener-io/ngrok#70a1a7d80d35931b97ddc3cde985599daac1ab68 postinstall /var/www/*******/client/node_modules/screener-runner/node_modules/ngrok
> node ./postinstall.js

module.js:549
    throw err;
    ^

Error: Cannot find module 'decompress-zip'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/var/www/********/client/node_modules/screener-runner/node_modules/ngrok/postinstall.js:5:11)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
npm WARN iqClient@0.0.1 No description
npm WARN iqClient@0.0.1 No repository field.
npm WARN iqClient@0.0.1 No license field.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.4 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.4: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: ngrok@2.2.27 (node_modules/screener-runner/node_modules/ngrok):
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: ngrok@github:screener-io/ngrok#70a1a7d80d35931b97ddc3cde985599daac1ab68 postinstall: `node ./postinstall.js`
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: Exit status 1
```

not sure why it can't find `decompress-zip` any idea?

**Comment by bubenshchykov** [comment]
hi, what are your node and npm versions?

**Comment by samirchian** [comment]
Hi, 
[root@ea81bcb644fc]# node -v
v8.10.0
[root@ea81bcb644fc]# npm -v
6.1.0

**Comment by bubenshchykov** [comment]
might be fixed in ngrok@3.1.1
